src/os/bluestore/bluestore_types.h

```diff
@@ -38,6 +38,7 @@ struct bluestore_bdev_label_t {
   string description;  ///< device description
 
   map<string,string> meta; ///< {read,write}_meta() content from ObjectStore
+  bool during_emergency_expand;
```

Comment:
May be it's better to keep the flag in meta map above and reuse read(write)_meta functions?